### DISCUSSION TITLE
MC Launcher Giving Error On Startup

### DISCUSSION THREAD

**Discussion by Swardcrafter** [FabricMC/fabric-api#1457] [Player support]
Hi ^ also the error is 
<img width="711" alt="Screen Shot 2021-05-16 at 9 46 13 AM" src="https://user-images.githubusercontent.com/84284643/118405401-9e903800-b62c-11eb-9b6a-5c9a0133cd12.png">

I only have Fabric API, Sodium, Phosphor, and Lithium running on 1.16.1 anyone have the fix?

**Comment by liach** [comment]
Do you have a crash report?
